Packs/WootCloud/Integrations/WootCloud/WootCloud_test.py

```diff
@@ -0,0 +1,164 @@
+
+
+def test_first_fetch_incidents(requests_mock, mocker):
+    client = Client(MOCK_URL, ssl_verify=True, api_version='/v1/', headers=MOCK_HEADERS, username='test_user', password='test123')
+    requests_mock.post(MOCK_URL + '/v1/events/packetalerts', json=FETCH_ALERTS)
+
+    fetch_incidents(client, 'packet')
```

Comment:
what is tested here? 
I think you should verify if fetch_incidents returns XSOAR incident properly